test/src/main/java/org/openjdk/skara/test/TestIssue.java

```diff
@@ -48,7 +48,7 @@ public class TestIssue implements Issue {
     protected final String title;
     protected final State state;
     // Mimic JiraIssue where labels are part of the main JSON object
```

Comment:
This comment is now outdated.
```suggestion
    // Labels are cached but still kept up to date
```